verification/cbmc/proofs/aws_cryptosdk_priv_hdr_parse_content_type/Makefile

```diff
@@ -0,0 +1,81 @@
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# if Makefile.local exists, use it. This provides a way to override the defaults
+sinclude ../Makefile.local
+#otherwise, use the default values
+include ../Makefile.local_default
+include ../Makefile.aws_byte_buf
+
+PROOF_UID = aws_cryptosdk_priv_hdr_parse_content_type
+
+HARNESS_ENTRY = $(PROOF_UID)_harness
+HARNESS_FILE = $(HARNESS_ENTRY).c
+
+# Values are chosen for performance. Increasing values does not improve coverage.
+MAX_EDK_LIST_ITEMS ?= 2
+MAX_TABLE_SIZE ?= 2
+
+DEFINES += -DARRAY_LIST_TYPE="struct aws_cryptosdk_edk"
+DEFINES += -DARRAY_LIST_TYPE_HEADER=\"aws/cryptosdk/edk.h\"
+DEFINES += -DAWS_NO_STATIC_IMPL
+DEFINES += -DMAX_EDK_LIST_ITEMS=$(MAX_EDK_LIST_ITEMS)
+DEFINES += -DMAX_TABLE_SIZE=$(MAX_TABLE_SIZE)
+
+CBMCFLAGS +=
+
+PROJECT_SOURCES += $(CBMC_ROOT)/aws-c-common/source/array_list.c
+PROJECT_SOURCES += $(CBMC_ROOT)/aws-c-common/source/byte_buf.c
+PROJECT_SOURCES += $(CBMC_ROOT)/aws-c-common/source/common.c
+PROJECT_SOURCES += $(CBMC_ROOT)/aws-c-common/source/hash_table.c
+PROJECT_SOURCES += $(CBMC_ROOT)/aws-c-common/source/string.c
+PROJECT_SOURCES += $(COMMON_PROOF_UNINLINE)/array_list.c
+PROJECT_SOURCES += $(COMMON_PROOF_UNINLINE)/error.c
+PROJECT_SOURCES += $(COMMON_PROOF_UNINLINE)/math.c
+PROJECT_SOURCES += $(COMMON_PROOF_UNINLINE)/string.c
+PROJECT_SOURCES += $(SRCDIR)/source/cipher.c
+PROJECT_SOURCES += $(SRCDIR)/source/edk.c
+PROJECT_SOURCES += $(SRCDIR)/source/enc_ctx.c
+PROJECT_SOURCES += $(SRCDIR)/source/header.c
+
+PROOF_SOURCES += $(COMMON_PROOF_SOURCE)/make_common_data_structures.c
+PROOF_SOURCES += $(COMMON_PROOF_SOURCE)/proof_allocators.c
+PROOF_SOURCES += $(COMMON_PROOF_SOURCE)/utils.c
+PROOF_SOURCES += $(COMMON_PROOF_STUB)/aws_array_list_defined_type.c
+PROOF_SOURCES += $(COMMON_PROOF_STUB)/aws_string_destroy_override.c
+PROOF_SOURCES += $(COMMON_PROOF_STUB)/error.c
+PROOF_SOURCES += $(COMMON_PROOF_STUB)/memcmp_override_no_op.c
+PROOF_SOURCES += $(PROOF_SOURCE)/make_common_data_structures.c
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE)
+
+# Remove function bodies to deal with CBMC function pointer removal
+# Functions removed for edk_list_clear
+REMOVE_FUNCTION_BODIES += --remove-function-body aws_array_list_get_at_ptr
+REMOVE_FUNCTION_BODIES += --remove-function-body nondet_compare
+
+# Functions removed for enc_ctx_clear
+REMOVE_FUNCTION_BODIES += aws_hash_callback_string_destroy
+REMOVE_FUNCTION_BODIES += aws_hash_string
+REMOVE_FUNCTION_BODIES += aws_raise_error_private
+REMOVE_FUNCTION_BODIES += aws_string_bytes
+REMOVE_FUNCTION_BODIES += aws_string_destroy
+REMOVE_FUNCTION_BODIES += aws_string_is_valid
+REMOVE_FUNCTION_BODIES += hashlittle2
```

Comment:
The correct variable is `REMOVE_FUNCTION_BODY`. With the starter kit, you do not need to specify the flag `--remove-function-body` anymore.

Reply:
Good to know! Updated for these proofs and going through the rest of the PRs now.